orbit/pkg/keystore/keystore_stub.go

```diff
@@ -0,0 +1,29 @@
+//go:build !(darwin && cgo) && !windows
+
+package keystore
+
+import "errors"
+
+func Supported() bool {
+	return false
+}
+
+func Name() string {
+	return "not implemented"
+}
+
+// AddSecret will add a secret to the keychain. This secret can be retrieved by this application without any user authorization.
```

Comment:
Not something we should change but in the past I've used the following pattern:

keystore.go:
```go
// AddSecret ...
func AddSecret(secret string) error {
  return addSecret(secret)
}
```
`keystore_windows.go`:
```go
func addSecret(secret string) error {
  // ...
}
```
`keystore_darwin.go`:
```go
func addSecret(secret string) error {
  // ...
}
```
`keystore_stub.go`:
```go
func addSecret(secret string) error {
  return errors.New("not implemented")
}
```

That way there's only one exported and documented method, and each implementation is kept private.